--- plot.R	2011-08-28 15:19:08.000000000 +0200
+++ plot-giw.R	2011-08-28 15:35:06.000000000 +0200
@@ -55,7 +55,7 @@
   abc <- sprintf('a = %.3g   b = %.3g   c = %g km', a, b, round(c))
 
 
-  yrange <- c(0, max(max(dif), a + b))
+  yrange <- c(a - b, max(max(dif), a + b))
   plot(xrange, yrange, xlab='Geograpic distance (km)', ylab='Linguistic difference', type='n', sub=abc)
   assign("needplot", FALSE, envir=.GlobalEnv)
   points(geo, dif, col='#A0A0A0', pch='.')
